components/ContentRenderer/ContentRenderer.module.scss

```diff
@@ -144,6 +144,10 @@ $avatar-size: 60px;
                     border-color: var(--prezly-border-color-secondary);
                 }
             }
+
+            &--wide {
+                width: 100% !important;
+            }
```

Comment:
I think we should fix this in the content renderer itself.

Reply:
I'll prepare a separate PR in the renderer